packages/esm-patient-tests-app/src/test-results/tree-view/tree-view-wrappper.component.test.tsx

```diff
@@ -0,0 +1,170 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import TreeViewWrapper from './tree-view-wrapper.component';
+import { useGetManyObstreeData } from '../grouped-timeline';
+import { useConfig } from '@openmrs/esm-framework';
+
+jest.mock('@openmrs/esm-framework', () => ({
+  useConfig: jest.fn(),
+  useLayoutType: jest.fn(),
+  createGlobalStore: jest.fn(),
+  createUseStore: jest.fn(),
+}));
```

Comment:
```suggestion
import { type ConfigObject } from '../config-schema';

const mockedUseConfig = jest.mocked(useConfig<ConfigObject>);
```

See https://o3-docs.openmrs.org/docs/frontend-modules/unit-and-integration-testing#mocking-patterns.